worker_server.py

```diff
@@ -41,19 +45,29 @@ def serve():
     server.wait_for_termination()
 
 if __name__ == '__main__':
-    parser = argparse.ArgumentParser(description = 'Please pass the port number and DelayFlag')
+    parser = argparse.ArgumentParser()
     parser.add_argument('PortNumber', help='Please pass the portNumber on which you want the worker to listen.')
     
     parser.add_argument("--addDelay", action="store_true", 
-                    help="Please pass the AddDelay parameter. This adds a random delay in worker operations.") 
+                    help="Please pass the AddDelay parameter. This adds a random delay in worker operations.")
+    
+    parser.add_argument('MaxWorkerCount', help='Please pass the max thread that you want worker to spawn.')
```

Comment:
Can we rename MaxWorkerCount since it seems ambiguous with Worker context that we have?

Reply:
Updated